src/libecalc/dto/components.py

```diff
@@ -293,7 +295,10 @@ class GeneratorSet(BaseEquipment):
     component_type: Literal[ComponentType.GENERATOR_SET] = ComponentType.GENERATOR_SET
     fuel: Dict[datetime, FuelType]
     generator_set_model: Dict[datetime, GeneratorSetSampled]
-    consumers: List[Union[ElectricityConsumer, ConsumerSystem]] = Field(default_factory=list)
+    consumers: List[Union[ElectricityConsumer, ConsumerSystem, Any]] = Field(
+        default_factory=list
+    )  # Any here is Pump, that cannot be explicitly specified due to circular import ...temporalequipment?
+    # Any must be set in order for pydantic to accept/validate the consumer
```

Comment:
How do we fix this?

Reply:
not a quick fix, but solve all the "hacks" where we have incorrect directions of dependencies.